spanner/test/spannerTest.php

```diff
@@ -931,11 +941,54 @@ public function testDmlReturningDelete()
      */
     public function testAddDropDatabaseRole()
     {
-        $output = $this->runFunctionSnippet('add_drop_database_role');
+        $output = $this->runFunctionSnippet('add_drop_database_role',
+        [self::$instanceId,
+        self::$databaseId,
+        self::$databaseRole]);
         $this->assertStringContainsString('Waiting for create role and grant operation to complete... ' . PHP_EOL, $output);
         $this->assertStringContainsString('Created roles new_parent and new_child and granted privileges ' . PHP_EOL, $output);
         $this->assertStringContainsString('Waiting for revoke role and drop role operation to complete... ' . PHP_EOL, $output);
-        $this->assertStringContainsString('Revoked privileges and dropped roles new_child and new_parent ' . PHP_EOL, $output);
+        $this->assertStringContainsString('Revoked privileges and dropped role new_child ' . PHP_EOL, $output);
```

Comment:
if you fix the above comment, probably this needs to be fixed as below:

```suggestion
        $this->assertStringContainsString('Revoked privileges and dropped role new_child' . PHP_EOL, $output);
```

Reply:
Done